avae/evaluate.py

```diff
@@ -10,15 +10,30 @@
 from .utils import set_device
 
 
-def evaluate(datapath, lim, splt, batch_s, collect_meta, use_gpu):
-
+def evaluate(
+    datapath,
+    state,
+    meta,
+    lim,
+    splt,
+    batch_s,
+    collect_meta,
+    use_gpu,
+    gaussian_blur,
+    normalise,
+    shift_min,
```

Comment:
Add to shift-mean to docstring.

Reply:
fixed in [9d23df5](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/9d23df5a61fa6f06f3200226aeeb560744fc04b9)